pkg/api/types.go

```diff
@@ -403,6 +404,13 @@ const (
 	PodUnknown PodPhase = "Unknown"
 )
 
+type PodCondition string
+
+// These are the valid service statuses of pods.
+const (
+	PodInService = "InService"
```

Comment:
NodeConditionStatus will be so similar to PodConditionStatus that NodeConditionStatus could be renamed to ConditionStatus and used for both cases, and eventually maybe for ContainerCondition. I don't think it would be detrimental to the api for some ConditionStatuses to be inapplicable to certain Condition types.